libs/libc/fdt/Kconfig

```diff
@@ -8,3 +8,13 @@ config LIBFDT
 	default n
 	---help---
 		Enable or disable Flattened Device Tree Library features
+
+if LIBFDT
+
+config DTC_VERSION
```

Comment:
maybe LIBFDT_DTC_VERSION is better ?

Reply:
I did not add LIBFDT prefix because libfdt is part of dtc. and it also used in version_gen.h in https://github.com/apache/nuttx-apps/pull/1916 which is defined DTC_VERSION

Reply:
As you mentioned, if libdft is part of dtc, why does libdef include the version of dtc? which confuse to me:

```
config LIBFDT
	bool "Flattened Device Tree Library"
	default n
	---help---
		Enable or disable Flattened Device Tree Library features

if LIBFDT

config DTC_VERSION
```

Reply:
let's change to LIBFDT_DTC_VERSION?